src/lib/comms/sol-oic-server.c

```diff
@@ -39,6 +39,12 @@
 #include "sol-oic-server.h"
 #include "sol-oic-security.h"
 
+#define STR(s) #s
+#define TO_STRING(s) STR(s)
+
+#define OIC_DATA_MODEL_VERSION "Unknown"
+#define OIC_SPEC_VERSION ""
```

Comment:
Why those options can not be configured in the build as the others ?

At least, check if it is alread defined before define.
```
#ifndef OIC_DATA_MODEL_VERSION
#define OIC_DATA_MODEL_VERSION
#endif
```